projects/igniteui-angular/src/lib/grids/tree-grid/tree-grid-expanding.spec.ts

```diff
@@ -366,6 +366,277 @@ describe('IgxTreeGrid - Expanding / Collapsing', () => {
 
 });
 
+describe('Row editing expanding/collapsing', () => {
+    configureTestSuite();
+    let fix;
+    let treeGrid;
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            declarations: [
+                IgxTreeGridRowEditingComponent
+            ],
+            imports: [
+                NoopAnimationsModule,
+                IgxTreeGridModule]
+        })
+        .compileComponents();
+    }));
+
+    beforeEach(() => {
+        fix = TestBed.createComponent(IgxTreeGridRowEditingComponent);
+        fix.detectChanges();
+        treeGrid = fix.componentInstance.treeGrid;
+        fix.detectChanges();
+    });
+
+    it('Hide banner with collapsing a node, using UI', fakeAsync(() => {
+        const rows = TreeGridFunctions.getAllRows(fix);
+
+        const cell = treeGrid.getCellByColumn(1, 'Name');
+        cell.inEditMode = true;
+        tick();
+        fix.detectChanges();
+
+        const firstRow = rows[0];
+        const indicatorDiv = TreeGridFunctions.getExpansionIndicatorDiv(firstRow);
+        indicatorDiv.triggerEventHandler('click', new Event('click'));
+        tick();
+        fix.detectChanges();
```

Comment:
Are tick() + detectChanges() necessary? triggerEventHandler should not require those.